conventional-commits.md

```diff
@@ -0,0 +1,132 @@
+`refactor` | 📦 Code Refactoring | A code change that neither fixes a bug nor adds a feature
+`perf` | 🚀 Performance Improvements | A code change that improves performance
+`test` | 🚨 Tests | Adding missing tests or correcting existing tests
+`build` | 🛠 Builds | Changes that affect the build system or external dependencies (example scopes: hatch, setuptool)
+`ci` | ⚙️ Continuous Integrations | Changes to our CI configuration files and scripts (example scopes: Travis, Circle, Github Actions)
+`chore` | ♻️ Chores | Other changes that don't modify src or test files
+`revert` | 🗑 Reverts | Reverts a previous commit
+
+## Description
+
+The description of the commit should be short and sweet,
+using imperative tense of description of the change.
+
+Action verbs like "fixes", "adds", "cleans" is great! See examples below.
+
+## Scope (*optional*)
+
+Scope adds a little more specificity to what this change relate to such as
+a component, module, or file name.
+
+## Body (*optional*)
+
+If you'd like to add a longer description of the change,
+you can provide it here. This allows for an extensive documentation of
+the reasons why the change happened.
+
+## Footer (*optional*)
+
+This allows for additional, specific description of the change.
+When there's a breaking change, it's best practice to add the `BREAKING CHANGE`
+footer followed by a description. Other footers can be some issue references,
+or reviewers. Footers should follow the [git trailer format](https://git-scm.com/docs/git-interpret-trailers), which is essentially: `token: value`.
+
+## Examples
+
+The following examples are retrieved directly from the conventional commit [docs](https://www.conventionalcommits.org/en/v1.0.0/#examples).
+
+### Commit message with description and breaking change footer
+
+```git
+feat: allow provided config object to extend other configs
+
+BREAKING CHANGE: `extends` key in config file is now used for extending other config files
+```
+
+### Commit message with `!` to draw attention to breaking change
+
+```git
+feat!: send an email to the customer when a product is shipped
+```
+
+### Commit message with scope and `!` to draw attention to breaking change
+
+```git
+feat(api)!: send an email to the customer when a product is shipped
+```
+
+### Commit message with both `!` and BREAKING CHANGE footer
+
+```git
+chore!: drop support for Node 6
+
+BREAKING CHANGE: use JavaScript features not available in Node 6.
+```
+
+### Commit message with no body
+
+```git
+docs: correct spelling of CHANGELOG
+```
+
+### Commit message with scope
+
+```git
+feat(lang): add Polish language
+```
+
+### Commit message with multi-paragraph body and multiple footers
+
+```git
+fix: prevent racing of requests
+
+Introduce a request id and a reference to latest request. Dismiss
+incoming responses other than from latest request.
+
+Remove timeouts which were used to mitigate the racing issue but are
+obsolete now.
+
+Reviewed-by: Z
+Refs: #123
```

Comment:
```suggestion
fix: prevent racing of requests

Introduce a request id and a reference to latest request. Dismiss
incoming responses other than from latest request.

Remove timeouts which were used to mitigate the racing issue but are
obsolete now.

---------

Reviewed-by: Z
Refs: #123
```

What do any of you think about adding a divider for the footer. Just saw that Github already does this if you co-author a PR in the PR commit message!

Reply:
Yes, that's a good idea to keep things neat.